test/ci/icarus_ci_intimecosmic_gen_quick_test_icaruscode.fcl

```diff
@@ -0,0 +1,27 @@
+#include "messages_icarus.fcl"
+#include "prodcorsika_proton_intime_icarus_bnb_sce_on_ovb.fcl"
+
+# non-interactive output settings, but keep them verbose
+services.message: @local::icarus_message_services_prod_debug
+
+# the test uses an input file (empty events with set timestamp)
+source: @erase
+
+# Be as strict as possible with random seeds: GENIE is wild!
+physics.producers.generator.Seed: 63214
+
+# Override GENIE flux file configuration
+physics.producers.generator.FluxCopyMethod: "DIRECT"
+physics.producers.generator.MaxFluxFileMB: 20
+physics.producers.generator.FluxSearchPaths: "/cvmfs/sbn.osgstorage.org/pnfs/fnal.gov/usr/sbn/persistent/stash/physics/beam/GENIE/BNB/standard/v01_00/"
+
```

Comment:
This looks like GENIE settings, which should not be in `CORSIKAGen` configuration.